src/python_bindings/bindings.cpp

```diff
@@ -160,7 +160,66 @@ PYBIND11_MODULE(desbordante, module) {
             .def("get_num_error_rows", &PyFDVerifier::GetNumErrorRows)
             .def("get_highlights", &PyFDVerifier::GetHighlights);
 
-    DEFINE_ALGORITHM(DataStats, Algorithm).def("get_result_string", &PyDataStats::GetResultString);
+    DEFINE_ALGORITHM(DataStats, Algorithm)
+            .def("get_all_statistics_as_string", &PyDataStats::GetAllStatisticsAsString)
+            .def("get_number_of_values", &PyDataStats::GetNumberOfValues,
+                 "Get number of values in the column.", py::arg("index"))
+            .def("get_number_of_columns", &PyDataStats::GetNumberOfColumns,
+                 "Get number of columns in the table.")
+            .def("get_null_columns", &PyDataStats::GetNullColumns,
+                 "Get indices of columns with only null values.")
+            .def("get_columns_with_null", &PyDataStats::GetColumnsWithNull,
+                 "Get indices of columns which contain null value.")
+            .def("get_columns_with_all_unique_values", &PyDataStats::GetColumnsWithUniqueValues,
+                 "Get indices of columns where all values are distinct.")
+            .def("get_number_of_distinct", &PyDataStats::Distinct,
+                 "Get number of unique values in the column.", py::arg("index"))
+            .def("is_categorical", &PyDataStats::IsCategorical,
+                 "Check if quantity => number of unique values in the column.", py::arg("index"),
```

Comment:
what does "quantity => number" means? Looks like a typo

Reply:
```suggestion
                 "Check if quantity >= number of unique values in the column.", py::arg("index"),
```

Reply:
I think "greater than" is better than ">="